native_client/ctcdecode/scorer.cpp

```diff
@@ -61,8 +61,11 @@ Scorer::Scorer(double alpha,
                const std::string& lm_path,
                const std::string& trie_path,
                const std::string& alphabet_config_path)
-  : Scorer(alpha, beta, lm_path, trie_path, Alphabet(alphabet_config_path.c_str()))
 {
+    Alphabet alphabet;
+    alphabet.loadConfigFile(alphabet_config_path.c_str());
```

Comment:
Hm. This is masking the failure when called from `DS_EnableDecoderWithLM`, which used to catch it with a try-catch block. The best thing to do would be to refactor `Scorer` as well to move fallible initialization to a method, but I'd also be happy with just adding a throw here if `loadConfigFile` fails.